packages/mutation-server-protocol/README.md

```diff
@@ -91,14 +91,16 @@ The `DiscoveredMutant` type is a subset of the `MutantResult` type. The `MutantR
 ```ts
 export interface DiscoverParams {
   /**
-   * The glob patterns to files to discover mutant in, or undefined to discover mutants in the entire project.
+   * The files to run discovery on, or undefined to discover all files in the current project.
```

Comment:
Undefined seems to me like a js/ts term. What does this mean in terms of the actual communication payload? Ommitted? Null? Empty array? All of the preceding?

Reply:
I would opt for using the term 'Ommitted'